routes/api.router.js

```diff
@@ -0,0 +1,8 @@
+const apiRouter = require("express").Router();
+const topicsRouter = require("./topics.router");
+
+
+apiRouter.use("/topics", topicsRouter);
```

Comment:
Great to see you using routers at this stage, it keeps the app file nice and tidy!